src/main/java/com/prgrms/web/api/v2/PostApiVer2.java

```diff
@@ -0,0 +1,95 @@
+package com.prgrms.web.api.v2;
+
+
+import static com.prgrms.dto.PostDto.Request;
+
+import com.prgrms.domain.post.PostService;
+import com.prgrms.dto.PostDto.Response;
+import com.prgrms.dto.PostDto.ResponsePostDtos;
+import com.prgrms.dto.PostDto.Update;
+import com.prgrms.global.error.ErrorCode;
+import com.prgrms.global.exception.AuthenticationFailedException;
+import com.prgrms.web.auth.SessionManager;
+import jakarta.servlet.http.HttpServletRequest;
+import java.net.URI;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/v2/posts")
+public class PostApiVer2 {
+
+    private final PostService service;
+    private final SessionManager sessionManager;
+
+    public PostApiVer2(PostService service, SessionManager sessionManager) {
+        this.service = service;
+        this.sessionManager = sessionManager;
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Response> getPostById(@PathVariable Long id) {
+
+        return ResponseEntity.ok(service.findPostById(id));
+    }
+
+    @PostMapping
+    public ResponseEntity<Void> registerPost(@RequestBody Request postDto,
+        HttpServletRequest request) {
+
+        long userId = sessionManager.getSession(request);
+
+        Response savedPost = service.insertPost(userId, postDto);
+        URI getPostByIdPath = URI.create("api/v2/posts/" + savedPost.getPostId());
```

Comment:
```suggestion
        URI resourceUri = URI.create("api/v2/posts/" + savedPost.getPostId());
```
로 추상화 된 표현을하는게 어떨까요?

Reply:
협업을 가정으로 두고 다른 개발자분이 이 코드를 읽어보셨을 때 어떤 api path인지 알수 있도록 
이렇게 변수명을 적어두었는데요, 추상화로 된 표현을 해야하는 이유를 여쭤봐도 될까요?
path가 변경될 수 있기 때문인걸까요?